install/migrations/update_10.0.x_to_11.0.0/tickettask.php

```diff
@@ -38,6 +38,10 @@
  */
 
 // new right values for task
-$migration->addRight('tickettask', CommonITILTask::ADDMYTICKET | CommonITILTask::ADDGROUPTICKET | CommonITILTask::ADD_AS_OBSERVER | CommonITILTask::ADD_AS_TECHNICIAN, [
-    'ticket' => Ticket::OWN
-]);
+$migration->updateRight(
+    'task',
+    CommonITILTask::ADDMYTICKET | CommonITILTask::ADDGROUPTICKET | CommonITILTask::ADD_AS_OBSERVER | CommonITILTask::ADD_AS_TECHNICIAN,
+    [
+        'ticket' => Ticket::OWN
+    ]
+);
```

Comment:
This will erase all other values. For instance, a user that have the `ticket/Ticket::OWN` right and the `task/CommonITILTask::SEEPRIVATE` will loose the `task/CommonITILTask::SEEPRIVATE` in this migration.

Reply:
This remark is still valid. The `Migration::updateRight()` does not handle this case as it is made to "replace" the value instead of "giving" the specified value.

Maybe the `Migration::updateRight()` should be renamed to `Migration::replaceRight()` (`Migration::updateRight()` should be only a proxy that triggers a deprecation) and a new `Migration::giveRight()` should be added.